e2e/app/component/share-modal.ts

```diff
@@ -14,11 +12,14 @@ export default class ShareModal extends Dialog {
   }
 
   async shareWithUser(username: string, level: WorkspaceAccessLevel): Promise<void> {
+    /*
     if (!this.page.viewport()) {
       throw Error(
         'testing of the share modal requires a viewport to be set on account ' +
           'on odd rendering issues with the react-select component');
     }
+    */
+
```

Comment:
works without setting viewport. I think finding element with `{visible: true}` is the reason.

Reply:
If that's the case, please remove the commented code! :)